integreat_cms/cms/views/organizations/organization_bulk_actions.py

```diff
@@ -0,0 +1,180 @@
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING
+
+from django.contrib import messages
+from django.utils.translation import gettext_lazy as _
+from django.utils.translation import ngettext_lazy
+
+from integreat_cms.cms.utils.stringify_list import iter_to_string
+
+from ...models import Organization
+from ..bulk_action_views import BulkActionView
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from django.http import HttpRequest, HttpResponse
+    from django.http.response import HttpResponseRedirect
+
+logger = logging.getLogger(__name__)
+
+
+class OrganizationBulkAction(BulkActionView):
+    """
+    View for executing organization bulk actions
+    """
+
+    #: The model of this :class:`~integreat_cms.cms.views.bulk_action_views.BulkActionView`
+    model = Organization
+
+
+class ArchiveBulkAction(OrganizationBulkAction):
+    """
+    Bulk action for archiving multiple organizations
+    """
+
+    def post(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
+        r"""
+        Function to archive multiple organizations at once
+
+        :param request: The current request
+        :param \*args: The supplied arguments
+        :param \**kwargs: The supplied keyword arguments
+        :return: The redirect
+        """
+        archive_successful = []
+        archive_failed = []
+
+        for content_object in self.get_queryset():
+            if content_object.archive():
+                archive_successful.append(content_object)
+            else:
+                archive_failed.append(content_object)
+
+        if archive_successful:
+            messages.success(
+                request,
+                ngettext_lazy(
+                    "{model_name} {object_names} was succesfully archived.",
+                    "The following {model_name_plural} were succesfully archived: {object_names}.",
+                    len(archive_successful),
+                ).format(
+                    model_name=self.model._meta.verbose_name.title(),
+                    model_name_plural=self.model._meta.verbose_name_plural,
+                    object_names=iter_to_string(archive_successful),
+                ),
+            )
+
+        if archive_failed:
+            messages.error(
+                request,
+                ngettext_lazy(
```

Comment:
Could you please explain this line? Is it not possible to use `_` here? :)

Reply:
Usually `_` is used in our code for singular messages (at least in `bulk_action_view`). Since singular isn't used here, `ngettext_lazy` could be imported as `_`.

Reply:
Thank you for explaining :)

Reply:
I think the plural use isn't in our documentation for internationalization yet

Reply:
Maybe importing `ngettext_lazy` as `_` isn't the best idea. It seems to mess with the translation file. I leave it as it is

Reply:
Alright :)